socs/agents/pysmurf_controller/agent.py

```diff
@@ -331,6 +331,8 @@ def _stop_check_state(self, session, params):
     @ocs_agent.param("duration", default=None, type=float)
     @ocs_agent.param('kwargs', default=None)
     @ocs_agent.param('load_tune', default=True, type=bool)
+    @ocs_agent.param('stream_type', default='obs')
```

Comment:
Could use the `choices=['obs', 'oper']` argument here to limit the possible inputs.

Reply:
I do vote for limiting the options here because that will prevent accidents / errors later down the chain.